frontend/src/components/universalComponents/ReportButton.tsx

```diff
@@ -0,0 +1,11 @@
+import {Link} from "react-router-dom";
+import React from "react";
+
+const ReportButton = () => {
+    return (
+        <div className="row d-flex justify-content-center">
+            <Link to="/report" className="my-3 btn btn-warning col-4">Un problème?</Link>
```

Comment:
btn-outline? Check if better

Reply:
i did, i dont like it not visible enough